src/test/java/gov/healthit/chpl/aqa/asserts/SurveillanceReportPageAsserts.java

```diff
@@ -119,8 +119,11 @@ public void verifyThatSurveillanceDataIsSavedInReport(final String friendlySurvI
         /* Workaround to resolve Stale Element Reference Exception-
          * reload the element after form update so it's attached to DOM: https://www.seleniumhq.org/exceptions/stale_element_reference.jsp*/
         getDriver().get(getUrl() + "#/surveillance/reporting");
+        getDriver().navigate().refresh();
         steps.iExpandACB("UL LLC");
-        SurveillanceReportPage.editSurveillanceReport(getDriver(), "UL LLC-2019-Q1").click();
+        WebElement editbutton = SurveillanceReportPage.editSurveillanceReport(getDriver(), "UL LLC-2019-Q1");
```

Comment:
This quarter isn't an editable quarter. That's fine to use it, but the PageElement indicates it's editable, not initiating, which is confusing

Has it been initiated in a previous step?

Reply:
Yes. It has been initiated in And I confirm initiating a quarterly surveillance report step.